fractions/swagger/src/main/resources/modules/com/fasterxml/jackson/swagger/module.xml

```diff
@@ -1,18 +1,17 @@
 <module xmlns="urn:jboss:module:1.3" name="com.fasterxml.jackson" slot="swagger">
   <resources>
-    <artifact name="com.fasterxml.jackson.core:jackson-core:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.core:jackson-annotations:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.core:jackson-databind:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-joda:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-guava:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-xml:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:${version.swagger.jackson}"/>
-    <artifact name="com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:${version.swagger.jackson}"/>
+    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-joda:${version.com.fasterxml.jackson}"/>
+    <artifact name="com.fasterxml.jackson.datatype:jackson-datatype-guava:${version.com.fasterxml.jackson}"/>
+    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${version.com.fasterxml.jackson}"/>
+    <artifact name="com.fasterxml.jackson.dataformat:jackson-dataformat-xml:${version.com.fasterxml.jackson}"/>
   </resources>
```

Comment:
As mentioned earlier, not all modules are available from WF 11. Looks like the datatype modules that are present in WF 11 point to a project that has been deprecated and needs to be fixed.